source/util.js

```diff
@@ -6,11 +6,13 @@ const execa = require('execa');
 const pMemoize = require('p-memoize');
 const ow = require('ow');
 
-exports.readPkg = () => {
-	const {packageJson} = readPkgUp.sync();
+exports.readPkg = packagePath => {
```

Comment:
@lukeggchapman This is what I mean:

```suggestion
exports.readPkg = (packagePath = process.cwd()) => {
```